certificate_manager/google_managed_lb_auth/main.tf

```
@@ -96,15 +96,13 @@ resource "google_compute_managed_ssl_certificate" "default" {
 
 # [START certificatemanager_google_managed_lb_auth_target_https_proxy]
 
-# Uncomment the `google_project.default` resource
-#data "google_project" "default" {
-#}
+data "google_project" "default" {
+}
 
 resource "google_compute_target_https_proxy" "default" {
   name = "test-proxy"
 
-  # Update the {certificate_map_name} and uncomment the certificate_map attribute
-  #  certificate_map = "//certificatemanager.googleapis.com/projects/${data.google_project.default.project_id}/locations/global/certificateMaps/{certificate_map_name}"
+  certificate_map = "//certificatemanager.googleapis.com/projects/${data.google_project.default.project_id}/locations/global/certificateMaps/certmap1"
```

nit: Based on https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/certificate_manager_certificate_map#id I think we could simplify this to
```suggestion
  certificate_map = "//certificatemanager.googleapis.com/${google_certificate_manager_certificate_map.default.id}"
```